functions/src/api/errors/HttpError.ts

```diff
@@ -54,6 +54,18 @@ export class ReminderNotFoundError extends NotFoundError {
     }
 }
 
+export class GoneError extends HttpError {
```

Comment:
Is this needed ? I find it not so clear

Reply:
Could be useful for other error but I'll remove the export